internal/pkg/log/log.go

```diff
@@ -29,15 +28,17 @@ func Init(configLogLevel string, logFile string, logInConsole bool) error {
 		return err
 	}
 
+	if logFile == "" {
+		logFile = "./testLogs.txt"
```

Comment:
for a better convention, logs file should be in .log file type such as:

controller.log
executor.log
test.log
and also you should put it inside some folder, for example, the logs folder will be in /var/log/octavius/controller.log

this would make our logging following standard UNIX scheme. Can you help refactor this?

Reply:
or we can create new folder in `~/.octavius/log/`

Reply:
Sure. Now all the logs are written in ~/.octavius/log/